block/manager.go

```diff
@@ -510,6 +510,16 @@ func (m *Manager) processNextDABlock(ctx context.Context) error {
 			}
 			m.logger.Debug("retrieved potential blocks", "n", len(blockResp.Blocks), "daHeight", daHeight)
 			for _, block := range blockResp.Blocks {
+				validBlock := block.SignedHeader.ValidateBasic() == nil
```

Comment:
why only `ValidateBasic` on signed header? why not the whole block? i am thinking what if the data is junk?